content/build/checks.md

```diff
@@ -224,3 +224,35 @@ directive in your Dockerfile or in a build argument. For example:
 ```console {title="Build argument"}
 $ docker build --check --build-arg "BUILDKIT_DOCKERFILE_CHECK=skip=JSONArgsRecommended,StageNameCasing;error=true" .
 ```
+
+## Experimental checks
+
+Before checks are promoted to stable, they may be available as experimental
+checks. Experimental checks are disabled by default.
+
+To enable all experimental checks, set the `BUILDKIT_DOCKERFILE_CHECK` build
+argument to `experimental=all`:
+
+```console
+$ docker build --check --build-arg "BUILDKIT_DOCKERFILE_CHECK=experimental=all" .
+```
+
+You can also enable experimental checks in your Dockerfile using the `check`
+directive:
+
+```dockerfile {title=Dockerfile}
+# syntax=docker/dockerfile:1
+# check=experimental=all
+```
+
+To selectively enable experimental checks, you can pass a CSV string of the
```

Comment:
It's probably worth specifying that because `experimental` is explicitly opt-in, it supersedes `skip` — something like`skip=all;experimental=CopyIgnoredFile` will still run the `CopyIgnoredFile` and `skip=all;experimental=all` will run all the `experimental` checks.

Reply:
Ok, yes that definitely needs a mention.

Reply:
done